utilities/backup.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
```

Comment:
Did you by chance update the permissions for this to be an executable? Unclear here if it has been set.